File: app/src/main/kotlin/de/ae/formulaecalendar/remote/pojo/driverstanding/ChampionshipStanding.java

package de.ae.formulaecalendar.remote.pojo.driverstanding;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ChampionshipStanding {

    @SerializedName("ChampionshipType")
    @Expose
    private String championshipType;
    @SerializedName("ChampionshipData")
    @Expose
    private ChampionshipData championshipData;

    public String getChampionshipType() {
        return championshipType;
    }

    public void setChampionshipType(String championshipType) {
        this.championshipType = championshipType;
    }

    public ChampionshipData getChampionshipData() {
        return championshipData;
    }

    public void setChampionshipData(ChampionshipData championshipData) {
        this.championshipData = championshipData;
    }

    @Override
    public String toString() {
        return "ChampionshipStanding{" +
                "championshipType='" + championshipType + '\'' +
                ", championshipData=" + championshipData +
                '}';
    }
}

File: app/src/main/kotlin/de/ae/formulaecalendar/remote/pojo/driverstanding/ChampionshipStanding.kt
package de.ae.formulaecalendar.remote.pojo.driverstanding

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ChampionshipStanding {

    @SerializedName("ChampionshipType")
    @Expose
    var championshipType: String? = null
    @SerializedName("ChampionshipData")
    @Expose
    var championshipData: ChampionshipData? = null

    override fun toString(): String {
        return "ChampionshipStanding{" +
                "championshipType='" + championshipType + '\'' +
                ", championshipData=" + championshipData +
                '}'
    }
}
